src/container.py

```diff
@@ -125,12 +156,38 @@ def _run_container(self, config: ComponentConfiguration):
         )
         self._follow_container_logs()
 
+    def _write_secrets_to_file(self, db_username, db_password):
+        db_user_path = self.secrets_path.joinpath(POSTGRES_USERNAME_FILE_KEY).resolve()
+        db_password_path_ = self.secrets_path.joinpath(POSTGRES_PASSWORD_FILE_KEY).resolve()
+        try:
+            # Create secrets directory if it
+            self.secrets_path.mkdir(parents=True, exist_ok=True)
+            # Write secret files
+            with open(db_user_path, "w") as u_file:
```

Comment:
What system user would write this files?  - If we are writing them to the files system could we make it read only to whatever role we are running greengrass as?

Reply:
this will be written by greengrass component user. That file is accessed by root and that greengrass component user only.